src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceImplIntTest_updateBirthPlace.kt

```diff
@@ -0,0 +1,110 @@
+@file:Suppress("ktlint:standard:filename", "ClassName")
+
+package uk.gov.justice.hmpps.prison.api.resource.impl
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderRepository
+
+class OffenderResourceImplIntTest_updateBirthPlace : ResourceTest() {
+
+  @Autowired
+  lateinit var offenderRepository: OffenderRepository
+
```

Comment:
You might want to use a builder rather then rely on seed data; e.g 
`      @BeforeAll
      fun setUp() {
        builder.build {
          prisoner = offender(lastName = "DUBOIS") {
            booking(
              prisonId = "MDI",
              bookingInTime = LocalDateTime.parse("2023-07-19T10:00:00"),
              movementReasonCode = REMAND_REASON,
            ) {}
          }
        }
      }`